src/liballoc/raw_vec.rs

```diff
@@ -555,6 +543,15 @@ unsafe impl<#[may_dangle] T, A: AllocRef> Drop for RawVec<T, A> {
     }
 }
 
+// Central function for reserve error handling.
+fn handle_reserve(result: Result<(), TryReserveError>) {
```

Comment:
Just wondering if this should this be `#[inline]`?

Reply:
I thought about that but I am not sure if it should be `#[inline]`, if it is not inlined then does it affect the stack trace?

Reply:
Not entirely sure regarding the stack trace (I don't think it really matters given that we're getting `#[track_caller]` soon). My reaction when seeing this is just that not inlining introduces another call when handling reservations. Given how frequently this code is called I thought it would probably be worth inlining to avoid that overhead.

Reply:
Okay, I added it.